user-access-event-handlers/src/test/java/no/unit/nva/useraccess/events/service/UserMigrationServiceTest.java

```diff
@@ -42,6 +43,7 @@
 class UserMigrationServiceTest {
 
     private static final URI SAMPLE_ORG_ID = URI.create("https://localhost/cristin/organization/123.0.0.0");
+    private static final URI SAMPLE_CRISTIN_API_ORG_ID = URI.create(CRISTIN_API_URI + randomString());
```

Comment:
if you use the host variable instead of URI this has to change I suppose to something like 

```
private static final URI SAMPLE_CRISTIN_API_ORG_ID =
        new UriWrapper("https",CRISTIN_API_URI).addChild(randomString()).getUri();
```